<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastGraph</class>
 <widget class="QDialog" name="FastGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>259</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fast Graph</string>
  </property>
  <widget class="FPlotter" name="fastPlot">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>478</width>
     <height>220</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>220</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>220</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QWidget" name="controls_widget" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>224</y>
     <width>355</width>
     <height>33</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSlider" name="gainSlider">
      <property name="toolTip">
       <string>Waterfall gain</string>
      </property>
      <property name="minimum">
       <number>-60</number>
      </property>
      <property name="maximum">
       <number>140</number>
      </property>
      <property name="pageStep">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="zeroSlider">
      <property name="toolTip">
       <string>Waterfall zero</string>
      </property>
      <property name="minimum">
       <number>-60</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="greenZeroSlider">
      <property name="toolTip">
       <string>Spectrum zero</string>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>160</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbAutoLevel">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>225</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set reasonable levels for gain and zero sliders.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Auto Level</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FPlotter</class>
   <extends>QFrame</extends>
   <header>fastplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
